Formula/claws-mail.rb

```diff
@@ -23,13 +23,19 @@ class ClawsMail < Formula
   depends_on "cairo"
   depends_on "glib"
   depends_on "gnutls"
-  depends_on "gtk+"
+  depends_on "gtk+3"
   depends_on "libetpan"
   depends_on "nettle"
 
+  # Fix compile on non-X11 platforms (macOS).
+  # https://www.thewildbeast.co.uk/claws-mail/bugzilla/show_bug.cgi?id=4505
+  patch do
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/a9cc349d382629ff1e5c805b6f699f0c4a65a0e3/claws-mail/4.0.0-macos.patch"
+    sha256 "36f37cf26343b2b5fe51b6a8b3aa28c5824ecd655109d9e8f6cd88d24b2d351f"
+  end
+
   def install
     system "./configure", "--prefix=#{prefix}",
-                          "LDFLAGS=-Wl,-framework -Wl,Security",
```

Comment:
I know this is currently making things fail, but this line is indicative of a problem with our libetpan - it's actually using the static library here which isn't ideal.

I'll fix it in the libetpan side.

Reply:
#96869 will hopefully fix it.